.azure-pipelines/pull_request.yml

```diff
@@ -93,6 +93,14 @@ stages:
               cd "$(build_dir)"
               cmake --build . --target install
             displayName: 'Build Python library'
+          - bash: |
+              set -ex
+              cd package/scipp-package-test
+              mkdir -p build && cd build
+              cmake -DCMAKE_PREFIX_PATH="$(install_dir)" ..
```

Comment:
Shouldn't multiple directories be semicolon separated?

Reply:
Multiple directories?

Reply:
On further inspection what you have is correct. Sorry assumed you were trying to include `..` in the prefix path.